src/xpanse-api/generated/models/Region.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+/**
+ * OpenAPI definition
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * OpenAPI spec version: v0
+ *
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { HttpFile } from '../http/http';
+
+/**
+ * The regions of the Cloud Service Provider
+ */
+export class Region {
+    'name'?: string;
```

Comment:
These fields should not be optional.
please generate the client after adding the OpenAPI validations which i mentioned in the Xpanse PR. 
After that.. these two properties in Region class will not be marked as optional.

Reply:
the field `name` is mandatory, but the `area` should be optional, I think.